components/file_system/src/metrics_task.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::sync::Arc;
+
+use tikv_util::IntervalRunnable;
+
+use crate::iosnoop::{fetch_io_bytes, flush_io_latency_metrics};
+use crate::metrics::IO_BYTES_VEC;
+use crate::BytesRecorder;
+use crate::IOBytes;
+use crate::{IOOp, IOType};
+
+pub enum BytesFetcher {
+    ByRateLimiter(Arc<BytesRecorder>),
```

Comment:
I assume this does not involve rate limit? If so rename it to ByBytesRecorder?

Reply:
Add some comment here about the difference between the two types?

Reply:
I'm inclined to keep the name because I plan to make limiter the owner of statistics and add toggle for recording (#9197).

Reply:
Made the change to reflect this thought.